applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -215,4 +215,32 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
+comment "BLE QoS management"
+
+config DESKTOP_BLE_QOS_ENABLE
+	bool "Enable BLE channel map management"
+	default n
+	depends on BT_CENTRAL
```

Comment:
This should probably depend on BT_LL_NRFXLIB.

Reply:
It does not depend on BT_CENTRAL

Reply:
Adding dependency here, and removing "#ifdef CONFIG_BT_LL_NRFXLIB" from ble_qos.c

Reply:
> 
> 
> It does not depend on BT_CENTRAL

For nrf_desktop it only makes sense using this feature on Central side, as the goal is to maintain an optimized channel map

Reply:
If this makes sense only on the central, then the central device should set `CONFIG_DESKTOP_BLE_QOS_ENABLE`. I suppose this library should be usable by any other application in the future, then we shouldn't arbitrarily limit it to `BT_CENTRAL`.